src/libsync/vfs/cfapi/cfapiwrapper.cpp

```diff
@@ -35,6 +36,7 @@ Q_LOGGING_CATEGORY(lcCfApiWrapper, "nextcloud.sync.vfs.cfapi.wrapper", QtInfoMsg
       FIELD_SIZE( CF_OPERATION_PARAMETERS, field ) )
 
 namespace {
+QMap<QString, QString> registeredSyncRootKeys;
```

Comment:
Please no non-POD static. This is always asking for trouble and especially here because the code in this file tends to be reached by different threads!

Besides (I still need to read the rest of this PR) but I'm so far kind of surprised you need this at all.

Reply:
@er-vin Then, I need to convert this file into a class instead of just a namespace. I need to store registry paths that I've created so I can then remove them when I need to unregister a sync root. Another way - is to add parameters for "unregister" function, but, then, I will also need to change the interface for `_vfs->unregister()`, just didn't want it to affect other platforms.

Reply:
No, no, you could go for Q_GLOBAL_STATIC (+ eventually a mutex to protect access if that's used on a code path from secondary thread, which in the context of that file would be anything called from one of the callbacks), it's having QMap straight as global static which will cause issues.

Reply:
> you could go for Q_GLOBAL_STATIC (+ eventually a mutex to protect access

@er-vin Done. Hopefully, this is what you had in mind?